.gitlab-ci.yml

```diff
@@ -42,6 +42,30 @@ test:
     paths:
       - coverage.lcov
 
+backwards-compat:
+  stage: test
```

Comment:
This is build only, right? Should it be named as such? Or moved to a different stage?

Reply:
I think it belongs to "test" stage in the sense that it tests backwards compatibility. We only want to make sure that it compiles (if the tests cannot run in old LTS we can live with that).

I will name it `test:backward-compat` for consistency with the test stage.